src/cli_shared/mod.rs

```diff
@@ -19,6 +20,14 @@ pub fn chain_path(config: &Config) -> PathBuf {
     PathBuf::from(&config.client.data_dir).join(config.chain.to_string())
 }
 
+/// Gets car db path
+pub fn car_db_path(config: &Config) -> anyhow::Result<PathBuf> {
+    let chain_data_path = chain_path(config);
+    let db_root_dir = db_root(&chain_data_path)?;
+    let forest_car_db_dir = db_root_dir.join("car_db");
```

Comment:
I've seen many occurrences of `car_db` in the code base, can we make it a constant?